### PR TITLE
add: focus event to textarea and error tag

### PR DESCRIPTION
**Pull request by GustavoUT22**

### CODE REVIEW

**Comment by Mardecera** [comment]
Input border jumps when in focus or active state

**Comment by Mardecera** [comment]
Textarea border jumps when in focus or active state

File: src/ui/Textarea/Textarea.tsx [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-  value?: string
-  onBlur?: (event: FocusEvent<HTMLTextAreaElement>) => void
-  onChange?: (event: ChangeEvent<HTMLTextAreaElement>) => void
-}
+import {
+  ErrorTextarea,
+  TextareaLabel,
+  TextareaWrapper,
+} from './Textarea.styles'
+import { IStyledTextarea } from 'types/ui/Textarea'
```
Comment (Mardecera) [inline src/ui/Textarea/Textarea.tsx:8]: IStyledTextarea or IPropsTextarea?

Comment (GustavoUT22) [reply src/ui/Textarea/Textarea.tsx]: done

**Review by Mardecera — APPROVED** [review]
Nice!

**Pull request merged**